### ISSUE TITLE
PROPOSAL: Create empty ./lg3.conf, iff missing?

### ISSUE THREAD

**Issue by HenrikBengtsson** [UCSF-Costello-Lab/LG3_Pipeline#126]
@ivan108 [wrote](https://github.com/UCSF-Costello-Lab/LG3_Pipeline/issues/125#issuecomment-484655048):

> may be an empty ./lg3.conf file should be created while running lg3 test setup?

**Comment by ivan108** [comment]
May be not completely empty, but with some comments in headers explaining what is this file for..?

**Comment by HenrikBengtsson** [comment]
So, the idea is to make it clearer to the end user that they can have a `./lg3.conf` file they can edit?

**Comment by ivan108** [comment]
YEs, it would be much easier to explain how to control runtime parameters with config file..
